app/src/main/java/com/codesoom/assignment/service/TodoPutService.java

```diff
@@ -0,0 +1,34 @@
+package com.codesoom.assignment.service;
+
+import com.codesoom.assignment.model.ResponseData;
+import com.codesoom.assignment.model.Task;
+import com.codesoom.assignment.repository.TaskRepository;
+import com.codesoom.assignment.repository.TaskRepositoryImpl;
+import com.codesoom.assignment.util.HttpStatus;
+import com.sun.net.httpserver.HttpExchange;
+
+import java.io.IOException;
+
+public class TodoPutService implements TodoService {
+    private final TaskRepository taskRepository = TaskRepositoryImpl.getInstance();
+    @Override
+    public ResponseData processRequest(HttpExchange exchange, Task taskParam, String pathVariable) throws IOException {
+        if (isUpdateRequest(pathVariable, taskParam)) {
+            taskParam.setId(Long.parseLong(pathVariable));
+
+            return taskRepository.findById(Long.parseLong(pathVariable)) != null ?
+                    new ResponseData(HttpStatus.HTTP_OK, convertToJSON(taskRepository.updateTask(taskParam))) :
+                    new ResponseData(HttpStatus.HTTP_NOT_FOUND, "");
+
+        } else {
+            return new ResponseData(HttpStatus.HTTP_OK, "");
+
+        }
```

Comment:
`else`문을 생략할 수 있다면 생략하도록 해봅시다.

```suggestion
        if (isUpdateRequest(pathVariable, taskParam)) {
            taskParam.setId(Long.parseLong(pathVariable));

            return taskRepository.findById(Long.parseLong(pathVariable)) != null ?
                    new ResponseData(HttpStatus.HTTP_OK, convertToJSON(taskRepository.updateTask(taskParam))) :
                    new ResponseData(HttpStatus.HTTP_NOT_FOUND, "");

        }
        return new ResponseData(HttpStatus.HTTP_OK, "");
```

Reply:
수정 반영했습니다.